challenge.md

```diff
@@ -31,22 +31,14 @@ Open up this file in your text editor, and write your answers below (where indic
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
- 
- <!-- Answer Ends Here -->
+It is a relative path because it is relative to a specific computer. Theoretically, this directory could exist on several computers. A longer, more specific path would be needed to access this directory remotely.
```

Comment:
This path is an Absolute path, because it provides a way to locate a particular resource that is _independent of your location_. Whether you're at `/`, `~`, or `/blah/blah/blah`, the path above will always send you to the exact same place. A Relative path, in contrast, defines where to go based on your location; as a result, it will point to different locations depending on where you are in the filesystem.